pkg/cmd/serviceaccount/create/create.go

```diff
@@ -144,6 +145,7 @@ func runCreate(opts *options) error {
 	creds := &credentials.Credentials{
 		ClientID:     serviceacct.GetClientId(),
 		ClientSecret: serviceacct.GetClientSecret(),
+		TokenURL:     "https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token",
```

Comment:
Perhaps as a build variable?

Reply:
Yes.. I was pulled to other work. Doing that now